src/inverseRobot.js

```diff
@@ -7,7 +7,33 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+  const massKeys = dublicate(Object.keys(robot));
+  const massValues = dublicate(Object.values(robot));
+
+  if (massKeys || massValues === null) {
```

Comment:
You should check 'massKeys === null' and 'massValues === null' separately using '||' operator.